src/remote-config/remote-config-api-client.ts

```diff
@@ -191,6 +191,36 @@ export class RemoteConfigApiClient {
       });
   }
 
+  public rollback(versionNumber: number | string): Promise<RemoteConfigTemplate> {
+    this.validateVersionNumber(versionNumber);
+    return this.getUrl()
+      .then((url) => {
+        const request: HttpRequestConfig = {
+          method: 'POST',
+          url: `${url}/remoteConfig:rollback`,
+          headers: FIREBASE_REMOTE_CONFIG_HEADERS,
+          data: { versionNumber: versionNumber }
+        };
+        return this.httpClient.send(request);
+      })
+      .then((resp) => {
+        if (!validator.isNonEmptyString(resp.headers['etag'])) {
```

Comment:
I'd assume this logic is used by at least one other method (getTemplate). Extract into a helper method?

Reply:
Good catch! Moved to a helper method. Thanks!